modules/fmops/bedrock-finetuning/bin/mlops_modules_development.ts

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+import "source-map-support/register";
+import * as cdk from "aws-cdk-lib";
+import { AmazonBedrockFinetuningStack } from "../lib/mlops_modules_development-stack";
+import "source-map-support/register";
+import * as cdk_nag from "cdk-nag";
+
+const account = process.env.CDK_DEFAULT_ACCOUNT;
+const region = process.env.CDK_DEFAULT_REGION;
+
+const projectName = process.env.SEEDFARMER_PROJECT_NAME;
+const deploymentName = process.env.SEEDFARMER_DEPLOYMENT_NAME;
+const moduleName = process.env.SEEDFARMER_MODULE_NAME;
+
+const vpcId: string | undefined = process.env.SEEDFARMER_PARAMETER_VPC_ID;
+const subnetIds: string[] = JSON.parse(
+  process.env.SEEDFARMER_PARAMETER_SUBNET_IDS || ("[]" as string),
+);
+const bedrockBaseModelID: string = process.env.BEDROCK_BASE_MODEL_ID!;
+
+const app = new cdk.App();
```

Comment:
Does this export any metadata i.e. input bucket, model?

Reply:
export of bucket name added